### PR TITLE
makes a mushroom to make you big

### PR DESCRIPTION
**Pull request by Linkr98**

### CODE REVIEW

File: curses.c [hunk 1]
```diff
@@ -77,15 +77,27 @@ int main(void)
     char tmp_board[ARRAY_SIZE][ARRAY_SIZE];
     // int sleep_time = 1000;
     int character_code;
-    int location_x = 5;
-    int location_y = 5;
+    int location_x = 5, location_y = 5, player_big = 0;
```
Comment (btanner) [inline curses.c]: I like that we have mush_x and mush_y. Do you think this is a good time to change location_x and location_y to just player_x and player_y?

**Review by btanner — CHANGES_REQUESTED** [review]

File: curses.c (hunk 1 above)
Comment (Linkr98) [reply curses.c]: done

File: curses.c [hunk 2]
```diff
@@ -9,7 +9,7 @@
 void copy_board(char dst[ARRAY_SIZE][ARRAY_SIZE], char src[ARRAY_SIZE][ARRAY_SIZE]);
 void clear_board(char some_board[ARRAY_SIZE][ARRAY_SIZE], char which_char);
 void print_board(char some_board[ARRAY_SIZE][ARRAY_SIZE]);
-void update(char current_board[ARRAY_SIZE][ARRAY_SIZE], char old_board[ARRAY_SIZE][ARRAY_SIZE]);
```
Comment (btanner) [inline curses.c]: You broke this. Fix it.

**Review by btanner — CHANGES_REQUESTED** [review]

File: curses.c (hunk 2 above)
Comment (Linkr98) [reply curses.c]: done!

**Comment by btanner** [comment]
Great work!

**Comment by btanner** [comment]
This resolves #7 !

**Pull request merged**